backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -99,18 +99,27 @@ public async Task<ActionResult<LexAuthUser>> AcceptEmailInvitation(RegisterAccou
         }
 
         var jwtUser = _loggedInContext.User;
-
-        var hasExistingUser = await _lexBoxDbContext.Users.FilterByEmailOrUsername(accountInput.Email).AnyAsync();
-        acceptActivity?.AddTag("app.email_available", !hasExistingUser);
-        if (hasExistingUser)
+        if (jwtUser.Email != accountInput.Email)
         {
-            ModelState.AddModelError<RegisterAccountInput>(r => r.Email, "email already in use");
+            // Someone is trying to reuse a JWT belonging to someone else. Naughty, naughty...
+            ModelState.AddModelError<RegisterAccountInput>(r => r.Email, "email address mismatch in invitation link");
```

Comment:
if we're going to make this change then we should update the UI to block the user from changing their email, otherwise it'll be very confusing that the frontend lets you change it while the backend does not

Reply:
chat with Chris, we would like to block changing the email on the frontend, with a note: "You can't change your email when accepting an invite, you can change your email after your account is registered, be aware this will invalidate any other invites you have received, so accept them first."